react/addressbook.jsx

```diff
@@ -18,6 +18,7 @@ var AddressBook = React.createClass({
       contactNames: [],
       currentPersonID: -1,
       editing: false,
+      photoUrl: "images/1.jpg",
```

Comment:
is there a nicer way to do this than magic strings? (actually naive question not a leading question)